core/utils/colour.js

```diff
@@ -92,7 +94,12 @@ Blockly.utils.colour.rgbToHex = function(r, g, b) {
  * @return {!Array.<number>} RGB representation of the colour.
  */
 Blockly.utils.colour.hexToRgb = function(hexColor) {
-  var rgb = parseInt(hexColor.substr(1), 16);
```

Comment:
@shakao hexToRgb assumes a hex color string as the input. I'm curious why you parse it again?